examples/jsm/loaders/FBXLoader.js

```diff
@@ -36,13 +36,15 @@ import {
 	Texture,
 	TextureLoader,
 	Uint16BufferAttribute,
+	Vector2,
 	Vector3,
 	Vector4,
 	VectorKeyframeTrack,
 	SRGBColorSpace
 } from 'three';
 import * as fflate from '../libs/fflate.module.js';
 import { NURBSCurve } from '../curves/NURBSCurve.js';
+import { Earcut } from '../../../src/extras/Earcut.js';
```

Comment:
Can you please use `ShapeUtils.triangulateShape()` instead of directly using `Earcut`? 

Sidenote: Importing the module directly from `src` can't be done because it breaks the usage with import maps.

Reply:
Ok, done.